cockroachcloud/egress-perimeter-controls.md

```diff
@@ -0,0 +1,335 @@
+
+    ~~~json
+    {
+      "name": "roach-buckets",
+      "type": "FQDN",
+      "destination": "storage.googleapis.com",
+      "ports": [
+        443,
+        80
+      ],
+      "paths": [
+        "/customer-manged-bucket-1/*",
+        "/customer-manged-bucket-2/*"
+      ],
+      "description": "egress for GCP storage buckets"
+    }
+    {
+      "name": "roach-kafka",
+      "type": "CIDR",
+      "destination": "123.34.62.123/32",
+      "ports": [
+        443
+      ],
+      "paths": [],
+      "description": "egress for Kafka"
+    }
+    ~~~
+
+1. Make the POST request to create each new rule.
+    
+    [API spec]
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    curl --request POST \
+    --header "Authorization: Bearer $CC_API_KEY" \
+    --url "https://management-staging.crdb.io/api/v1/clusters/$CLUSTER_ID/networking/egress-rules" \
+    --data "@egress-rule1"
+
+    curl --request POST \
+    --header "Authorization: Bearer $CC_API_KEY" \
+    --url "https://management-staging.crdb.io/api/v1/clusters/$CLUSTER_ID/networking/egress-rules" \
+    --data "@egress-rule2"
+    ~~~
+
+    ~~~json
+
+    ~~~
+
+    {{site.data.alerts.callout_danger}}
+    Your cluster's firewall behavior is not updated instantly when you submit the API request. After, submitting the request, [check your egress rules](#check-a-clusters-egress-rules-allowed-destinations) to confirm that the new rules have been created.
+    {{site.data.alerts.end}}
+
+### Check the status of a rule
+
+{{site.data.alerts.callout_info}}
+Consult the glossary of [rule statuses](#rule-statuses).
+{{site.data.alerts.end}}
+
+[API spec]
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+curl --request GET \
+  --url 'https://cockroachlabs.cloud/api/v1/clusters/$CLUSTER_ID/networking/egress-rules' \
+  --header "Authorization: Bearer $CC_API_KEY"
+~~~
+
+~~~txt
+
+~~~
+
+### Check a cluster's egress rules/allowed destinations
+
+[API spec]
+
+{{site.data.alerts.callout_info}}
+Consult the glossary of [rule statuses](#rule-statuses).
+{{site.data.alerts.end}}
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+curl --request GET \
+  --url 'https://cockroachlabs.cloud/api/v1/clusters/$CLUSTER_ID/networking/egress-rules' \
+  --header "Authorization: Bearer $CC_API_KEY"
+~~~
+
+~~~txt
+
+~~~
+
+### Removing a rule
+
+A `DELETE` request to the rule's path tells the API to delete the rule.
+
+{{site.data.alerts.callout_danger}}
+Your cluster's firewall behavior is not updated instantly when you submit the API request. After, submitting the request, [check your egress rules](#check-a-clusters-egress-rules-allowed-destinations) to confirm that the deletion is complete.
+{{site.data.alerts.end}}
+
+{% include_cached copy-clipboard.html %}
+~~~shell
+curl --request DELETE \
+  --url 'https://cockroachlabs.cloud/api/v1/clusters/$CLUSTER_ID/networking/egress-rules/$RULE_ID' \
+  --header "Authorization: Bearer $CC_API_KEY"
+~~~
+
+~~~txt
+API RESPONSE!
```

Comment:
TODO?

Reply:
A full `EgressRule` is returned in the delete response.